app/api/models/statistics.py

```diff
@@ -54,6 +104,10 @@ class GenomeStatistics(BaseModel):
     assembly_stats: Assembly
     coding_stats: Coding
     variation_stats: Variation
+    none_coding_stats: NoneCoding
```

Comment:
`none_coding ` -> `non_coding` (both in the variable name and in the class name)